android/build.gradle

```diff
@@ -31,5 +31,5 @@ dependencies {
     compile 'com.facebook.react:react-native:0.20.+'
     compile fileTree(include: ['*.jar'], dir: 'libs')
     testCompile 'junit:junit:4.12'
-    compile 'com.github.afollestad.material-dialogs:commons:0.8.6.2'
+    compile 'com.afollestad.material-dialogs:commons:0.8.6.2
```

Comment:
Whoops, my bad. Guess that's what happens when I build out of one repo and make PRs out of another.